supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -108,6 +108,18 @@
 
 *See also*:
 
+[[offline]]
+== image:images/yes.png[yes] offline (adjective)
+
+*Description*: In Red{nbsp}Hat build of MicroShift, use for endpoints where no network connection is present and/or all network cards are unplugged. Examples include edge devices in physically remote locations. For situations where a network is present, but devices or hosts are isolated from it, such as in restricted networks or air-gapped networks, use _disconnected_.
```

Comment:
Per IBM Style, _and/or_ is a "do not use" term. Would something like this revision work? 

`In Red{nbsp}Hat build of MicroShift, use for endpoints where no network connection is present, all network cards are unplugged, or where no connection is present and all network cards are unplugged.`

![Screenshot 2024-06-27 at 12 53 53 PM](https://github.com/redhat-documentation/supplementary-style-guide/assets/102675557/b8368679-bcc9-42bc-960a-a2e9653e620a)

Reply:
I split the sentences up. WDYT?